src/templates/_macros/app_tile.html

```diff
@@ -57,6 +60,13 @@
   </{{ tag }}>
 
   {% if is_detail %}
+    {% if app.isWebsite %}
```

Comment:
At mobile widths it is beside it.

![screenshot 2015-05-15 14 36 07](https://cloud.githubusercontent.com/assets/211578/7660456/c57a6c44-fb0f-11e4-9576-9ec6bae34332.png)